### PR TITLE
OP-507 more user guide updates

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-507

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -172,29 +172,29 @@ Press the *[.underline]##N##ew* button in the *_OPD Out Patient Department_* win
...
-The second and third list still contains all diseases.
+* *Admittance Date*. The date on which the patient is seen for the recorded activity.
+* *Disease Type:* By selecting a _Disease Type_, the first diagnosis list will contain only related diseases.
+The second and third list continues to contain all diseases.
  +
- +
-_Example: Disease Types can be defined by the Administrator. Ask your Administrator how to do it or see
+
+NOTE: Disease Types are defined by the Administrator. Ask your Administrator how to do it or see
 <<settings,Settings>> in this document._
```
Comment (mwithi) [inline doc_user/UserManual.adoc:196]: is the trailing underscore correct?

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -341,45 +340,45 @@ To insert a new pharmaceutical, enter the following fields:
 * *Type* (for instance, Drugs or Chemical) +
 * *Code*: code that uniquely identifies the pharmaceutical for fast searching
 * *Description*: the description of the pharmaceutical
-* *PcsXPck*: Pieces per packet (if more than 1)
+* *PcsXPck*: Piece per packet (if more than 1)
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: should not be plural?

**Review by mwithi — APPROVED** [review]
Just two comments! Thanks a lot!

File: doc_user/UserManual.adoc (hunk 1 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc:196]: Nope; corrected; thanks

File: doc_user/UserManual.adoc (hunk 2 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Yes, put  not uppercased :) thanks

**Pull request merged**